### ISSUE TITLE
[Suggestion] Add github authentication on login page

### ISSUE THREAD

**Issue by kchatpar** [CONP-PCNO/conp-authentication#5]
Consider adding a Github sign in button on the login page to be able to track developer's projects. The reason is that you can track how developers are using the data and the outcome can be tracked.

**Comment by shots47s** [comment]
My personal opinion would be no, usually you add this for a page that involves development, so I this is unnecessary, but I am willing to be convinced.

**Comment by jbpoline** [comment]
agreed - this does not seem necessary given the targetted community

**Comment by driusan** [comment]
Given that people who don't authenticate using edugain need to be separately approved, I'd support adding support for it. The more login backends supported the easier it is for anyone to access.

**Comment by DESm1th** [comment]
I agree with driusan. Also even if users choose to use a different authentication method if users choose to supply their github username info about their public repos and stuff from could be pulled from the github API

**Comment by shots47s** [comment]
Ok, I am convinced :)

-- 
Shawn T. Brown, PhD.
Associate Director of Research Software Development  / CBRAIN Platform
McGill Centre for Integrative Neuroscience (MCIN)
McGill University
Montreal, QC, Canada
Email: stbrown@mcin.ca
Phone: 514-237-2960
